functions/find.bash

```diff
@@ -90,7 +90,7 @@ find3_setup() {
   if ! cond_redirect cd $find3Dir/server/ai; then echo "FAILED (cd)"; return 1; fi
   if cond_redirect python3 -m pip install -r requirements.txt; then echo "OK"; else echo "FAILED (pip)"; return 1; fi
 
-  if ! zram_dependency install find3server find3ai; then return 1; fi
+  if ! cond_redirect zram_dependency install find3server find3ai; then return 1; fi
```

Comment:
same here please don't mix everything into this CI PR

Reply:
I mixed these in because I discovered that the output was funky without the `cond_redirect`